RCSwitch.cpp

```diff
@@ -457,6 +477,64 @@ char* RCSwitch::getCodeWordD(char sGroup, int nDevice, boolean bStatus){
 }
 
 /**
+ * Get coding for Type E (RSL devices sold by Conrad)
+ * source: http://www.mikrocontroller.net/topic/252895
```

Comment:
I'm confused: that posting is from 2012, and one of the people posting there,  Tomat3007, writes that he added this protocol to RCSwitch back then. Huh?